draft-ietf-masque-h3-datagram.md

```diff
@@ -474,6 +474,18 @@ reencode HTTP Datagrams into QUIC DATAGRAM frames over the next hop, and those
 could be dropped. Because of this, applications have to always consider HTTP
 Datagrams to be unreliable, even if they were initially sent in a capsule.
 
+If an intermediary receives an HTTP Datagram in a QUIC DATAGRAM frame and is
+forwarding it on a connection that supports QUIC DATAGRAM frames, the
+intermediary SHOULD NOT convert that HTTP Datagram to a DATAGRAM capsule. If
+the HTTP Datagram is too large to fit in a DATAGRAM frame because the MTU of
```

Comment:
Can we avoid using MTU here? https://www.rfc-editor.org/rfc/rfc9000.html#section-14 is relevant, unfortunately it calls it "Datagram size" which is a world of hurt. But the important thing is that the forwarding is limited by the the peer jut as much as the path. 

> Enforcement of the max_udp_payload_size transport parameter (Section 18.2) might act as an additional limit on the maximum datagram size.

Would "maximum UDP payload size (Section 18.2; RFC 9000)" work as a replacement.

Reply:
Can you clarify why would we want to avoid using MTU?

Reply:
Mainly because all of the hoops that RFC 9000 jumps through in order to spell out considerations for QUIC packet sizes beyond just MTU.

In the simplest sense, if an operator has configured a network in such a way to believes that MTU of the single hop is 1600 but the QUIC endpoints both advertise a max_udp_payload_size TP of 1400, they might be surprised when the larger size isn't used.

Reply:
That makes sense. How about phrasing it as examples (see latest commit)

Reply:
great, ship it!